.github/workflows/ci.yaml

```diff
@@ -190,7 +196,3 @@ jobs:
       - uses: actions/checkout@v2
       - name: Codespell test
         uses: codespell-project/actions-codespell@master
-        with:
-          check_filenames: true
-          check_hidden: true
-          skip: ./.git,go.sum
```

Comment:
is it ok to remove this?  i didnt see any codespell configs in the infra-checkers repo.

Reply:
you are right, thank you!

Reply:
Restored here: f73c416eef0f2de4b845b6722c3278960eb57fd6